platform/ios/PolyPodApp/Features/Feature.swift

```diff
@@ -11,35 +12,57 @@ class Feature {
     let thumbnail: URL?
     private let links: [String: String]
     
-    static func load(path: URL, languageCode: String?) -> Feature? {
-        let manifest = readManifest(path)
+    static func load(path: URL) -> Feature? {
+        guard let manifest = readManifest(path) else {
+            return nil
+        }
         return Feature(
             path: path,
-            manifest: manifest,
-            languageCode: languageCode
+            manifest: manifest
         )
     }
     
-    init(path: URL, manifest: FeatureManifest, languageCode: String?) {
+    init(
+        path: URL,
+        name: String?,
+        author: String?,
+        description: String?,
+        thumbnail: String?,
+        thumbnailColor: String?,
+        primaryColor: String?,
+        links: [String: String]?
```

Comment:
That makes for a pretty terrible signature - and imagine all the cascading changes we'd have to make if we were to add or remove something from the manifest. Can't we keep a data class for this? Ideally I'd expect core to provide us one.

Reply:
But if you were to just pass in a `FeatureManifest` object like you did before, you'd have even _fewer_ changes in the `Feature` class - that's my point. I know we don't get a neat `FeatureManifest` object from core, but that's one of my main points: I think we should get one from _PlatformCoreSwift_ - dealing with the actual data coming from platform core is boiler plate, why not encapsulate that boiler plate into the wrapper library?

Reply:
This boilerplate is added only because of how Feature is used in PolyPod. A particular case is creating a mock Feature for preview. I didn't want to spin-up the whole flat buffer logic just to create a mock for the Feature.

I don't want to insist more than needed, but I would be more conformable refactoring how `FeatureManifest` is consumed in the UI in a separate PR.